From b54ef297b3c723c8438596c6e6afef93a7458b98 Mon Sep 17 00:00:00 2001
From: Taylor Otwell <taylorotwell@gmail.com>
Date: Mon, 9 Nov 2020 14:36:40 -0600
Subject: [PATCH] add auth line

---
 resources/lang/en/auth.php | 1 +
 1 file changed, 1 insertion(+)

diff --git a/resources/lang/en/auth.php b/resources/lang/en/auth.php
index e5506df290..6598e2c060 100644
--- a/resources/lang/en/auth.php
+++ b/resources/lang/en/auth.php
@@ -14,6 +14,7 @@
     */
 
     'failed' => 'These credentials do not match our records.',
+    'password' => 'The provided password is incorrect.',
     'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
 
 ];
