From 5182e9c6de805e025fb4cfad63c210c3197002ab Mon Sep 17 00:00:00 2001
From: Taylor Otwell <taylorotwell@gmail.com>
Date: Tue, 17 Nov 2020 17:07:32 -0600
Subject: [PATCH] add ably entry

---
 config/broadcasting.php | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/config/broadcasting.php b/config/broadcasting.php
index 3bba1103e6..ef20859859 100644
--- a/config/broadcasting.php
+++ b/config/broadcasting.php
@@ -41,6 +41,11 @@
             ],
         ],
 
+        'ably' => [
+            'driver' => 'ably',
+            'key' => env('ABLY_KEY'),
+        ],
+
         'redis' => [
             'driver' => 'redis',
             'connection' => 'default',
