From bc339f712389cf536ad7e340453f35d1dd865777 Mon Sep 17 00:00:00 2001
From: Taylor Otwell <taylorotwell@gmail.com>
Date: Wed, 16 Dec 2020 08:44:41 -0600
Subject: [PATCH] add lock_connection

---
 config/cache.php | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/cache.php b/config/cache.php
index 9473acc9845..963adfb564d 100644
--- a/config/cache.php
+++ b/config/cache.php
@@ -46,6 +46,7 @@
             'driver' => 'database',
             'table' => 'cache',
             'connection' => null,
+            'lock_connection' => null,
         ],
 
         'file' => [
@@ -75,6 +76,7 @@
         'redis' => [
             'driver' => 'redis',
             'connection' => 'cache',
+            'lock_connection' => 'cache',
         ],
 
         'dynamodb' => [
