From ed53ce54b19b6b726019f7bb6f80516db4ac3e0f Mon Sep 17 00:00:00 2001
From: Jared Lewis <jslewis90@outlook.com>
Date: Tue, 9 Feb 2021 11:27:26 -0500
Subject: [PATCH] Add unverified state to UserFactory

---
 database/factories/UserFactory.php | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/database/factories/UserFactory.php b/database/factories/UserFactory.php
index bdea1a32dc..99dfdcb6d4 100644
--- a/database/factories/UserFactory.php
+++ b/database/factories/UserFactory.php
@@ -30,4 +30,18 @@ public function definition()
             'remember_token' => Str::random(10),
         ];
     }
+
+    /**
+     * Define the model's unverified state.
+     *
+     * @return \Illuminate\Database\Eloquent\Factories\Factory
+     */
+    public function unverified()
+    {
+        return $this->state(function (array $attributes) {
+            return [
+                'email_verified_at' => null,
+            ];
+        });
+    }
 }
